spec/models/calculator_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Calculator, type: :model do
+  before { build(:calculator) }
+  describe 'associations' do
+    it { should have_many(:fields)}
+  end
+
+  describe 'validations' do
+    it { should validate_presence_of(:name) }
+    it { should validate_length_of(:name).
+              is_at_least(2).
+              on(:create) }
+    it { should_not allow_value("Hh34").for(:name) }
+  end
```

Comment:
Do not use `should` in your tests. Prefer `expect` or `is_expected`

Reply:
Thanks. I've changed it in my last commit.